app/js/main.js

```diff
@@ -3,6 +3,7 @@ const LOAD_EVENT = 'load';
 
 const TRUE_TEXT = 'true';
 const LIST_TAG = 'li';
```

Comment:
LIST_TAG is unused.

Reply:
Addressed in [15bd39f](https://github.com/markuszeller/image-stitcher/pull/3/commits/15bd39fe476611459db9e09fb9e23d6644a2eae0)